# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xautolock/wait-hotfix.patch
# Copyright (C) 2020 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xautolock-2.2/src/engine.c.vanilla	2020-02-20 20:50:38.027123423 +0000
+++ xautolock-2.2/src/engine.c	2020-02-20 20:51:12.091123247 +0000
@@ -211,7 +211,7 @@
   if (lockerPid)
   {
 #if !defined (UTEKV) && !defined (SYSV) && !defined (SVR4)
-    union wait  status;      /* childs process status */
+    int status;      /* childs process status */
 #else /* !UTEKV && !SYSV && !SVR4 */
     int         status = 0;  /* childs process status */
 #endif /* !UTEKV && !SYSV && !SVR4 */
